external/contracts/set/BasicIssuanceModule.sol

```diff
@@ -183,11 +199,37 @@ contract BasicIssuanceModule is ModuleBase, ReentrancyGuard {
     }
 
     /**
-     * Reverts as this module should not be removable after added. Users should always
-     * have a way to redeem their Sets
+     * SET TOKEN ONLY: Allows removal (and deletion of state) of BasicIssuanceModuleV2
      */
     function removeModule() external override {
-        revert("The BasicIssuanceModule module cannot be removed");
+        require(issuanceSettings[ISetToken(msg.sender)].moduleIssuanceHooks.length == 0, "Registered modules must be removed.");
+        delete issuanceSettings[ISetToken(msg.sender)];
+    }
+
+    /**
+     * MANAGER ONLY: Updates the address of the manager issuance hook. To remove the hook
+     * set the new hook address to address(0)
+     *
+     * @param _setToken         Instance of the SetToken to update manager hook
+     * @param _newHook          New manager hook contract address
+     */
+    function updateManagerIssuanceHook(
```

Comment:
this functionality is new. Not sure if really needed. Could be excluded because we have module hooks?

Reply:
We do use the `supplyCapHook` which is a manager issuance hook, so could be good to have a way to update it.